tests/FeedGenerator/FeedGeneratorTest.php

```
@@ -118,16 +118,20 @@ private function getAtomFeed(Blog $blog, array $posts): SimpleXMLElement
     {
         $feedGenerator = $this->createFeedGenerator();
         $feedData = $feedGenerator->generateAtomFeed($blog, $posts);
+        /** @var SimpleXMLElement $result */
+        $result = simplexml_load_string($feedData);
```

Did we not ignore this error?

---

No that was a different one, also todo with simpleXML